Software/GuitarPedal/Effect-Modules/Nam/model_data_nam.h

```diff
@@ -0,0 +1,792 @@
+struct modelDataNam {
+    std::vector<float> weights;
+};
+
+// ADD YOUR MODEL IDENTIFIER HERE ////////////////////////////////// < -------------------
+modelDataNam NamModel1;
+modelDataNam NamModel2;
+modelDataNam NamModel3;
+modelDataNam NamModel4;
+modelDataNam NamModel5;
+modelDataNam NamModel6;
+modelDataNam NamModel7;
+modelDataNam NamModel8;
+
+/////////////////////////////////////////////////////////////////
+
+std::vector<modelDataNam> model_collection_nam;
+
+/*========================================================================*/
+
+void setupWeightsNam() {
+
+    // COPY AND PASTE YOUR MODEL WEIGHTS BELOW (After converting .json to .h file) ////////////////////////////////// <
+    // -------------------
+    //   ADD AND REMOVE MODELS AS DESIRED (CAN HOLD AROUND 15-16 MODELS IN FLASH MEMORY)
+
+    //========================================================================
+    // NAM Pico Models
+    /*
+    {"version": "0.5.4", "metadata": {"date": {"year": 2024, "month": 12, "day": 28, "hour": 10, "minute": 21, "second": 12},
+    "loudness": -12.713208198547363, "gain": 0.6453793642123751}, "architecture": "WaveNet", "config": {"layers": [{"input_size": 1,
+    "condition_size": 1, "head_size": 2, "channels": 2, "kernel_size": 3, "dilations": [1, 2, 4, 8, 16, 32, 64], "activation": "Tanh",
+    "gated": false, "head_bias": false}, {"input_size": 2, "condition_size": 1, "head_size": 1, "channels": 2, "kernel_size": 3,
+    "dilations": [128, 256, 512, 1, 2, 4, 8, 16, 32, 64, 128, 256, 512], "activation": "Tanh", "gated": false, "head_bias": true}],
+    "head": null, "head_scale": 0.02},
+    */
+
```

Comment:
I probably could/should have used `//clang-format off` and `//clang-format on` for these blocks but I think it is ok for them to be nicely formatted as well